File: templates/docs-structure/examples/node/perfect-schema.ts
// EXAMPLE — canonical pattern, copy & adapt.
// A shared zod schema + inferred type.
import { z } from 'zod';

export const User = z.object({
  id: z.string().uuid(),
  email: z.string().email(),
  createdAt: z.string().datetime(),
});

export type User = z.infer<typeof User>;

File: templates/docs-structure/examples/python/perfect_schema.py
"""EXAMPLE — canonical pattern, copy & adapt.

A pydantic schema = typed data contract at a boundary.
"""
from __future__ import annotations

from datetime import datetime
from uuid import UUID

from pydantic import BaseModel, EmailStr


class User(BaseModel):
    id: UUID
    email: EmailStr
    created_at: datetime
